### ISSUE TITLE
captcha.php (CzaksCaptcha) issues

### ISSUE THREAD

**Issue by frozenpandaman** [vichan-devel/vichan#620]
I set my database info correctly in `inc/captcha/config.php` and my `inc/instance-config.php` includes the following lines, per the readme.md instructions:
```php
$config['captcha']['enabled']        = true;
$config['additional_javascript'][]   = 'js/captcha.js';
$config['captcha']['provider_get']   = '../inc/captcha/entrypoint.php';
$config['captcha']['provider_check'] = '../inc/captcha/entrypoint.php';
```
But the verification field just shows the "click me" image and nothing happens when I click it – no captcha loads or shows up.

vichan 5.1.4

**Comment by ctrlcctrlv** [comment]
open js console and review the log of the js xhr sent when you click it

**Comment by frozenpandaman** [comment]
Thank you both!

@ctrlcctrlv, nothing was appearing in the console when clicking on the default `static/clickme.gif`, even though `/js/captcha.js` was indeed being loaded. And @crazy4cars69, it doesn't seem to actually need the full url, it's fine beginning with `../inc/`.

However when I tried to change it to a full URL, I typed it wrong, and that led to a 404 GET being logged… but navigating to the correct `/inc/captcha/entrypoint.php?mode=get&extra=abcdefghijklmnopqrstuvwxyz` showed me tons of lines of
```
<b>Deprecated</b>:  Implicit conversion from float 22.666666666666668 to int loses precision in <b>/mypath/inc/captcha/captcha.php</b> on line <b>44</b><br />
<br />
<b>Deprecated</b>:  Implicit conversion from float 34.66666666666667 to int loses precision in <b>/mypath/inc/captcha/captcha.php</b> on line <b>44</b><br />
```
etc… preceding the expected JSON response.

So I fixed this by casting it to an int and it seems to work now! I'll make a PR to fix this.
